README.md

```diff
@@ -1 +1,37 @@
-# path-utils
\ No newline at end of file
+# glob-path-utils
+
+Expands paths to files by mask
```

Comment:
Wrapper for utility [glob]() which provides expanding of masks, dirs and files to absolute paths.